### DISCUSSION TITLE
[code-links] Support devcontainer.json in sub folder

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#6683] [Feature Requests]
### Description

I am excited to see the Repo link feature trying to support Dev Container / GitHub Codespaces. c7803eab16a217cc3fd26a2039db7c090cedd8f0

By the way, note that multiple Dev Container definitions can be included in a single repository.
The Spec says:

> - .devcontainer/devcontainer.json
> - .devcontainer.json
> - .devcontainer/\<folder\>/devcontainer.json (where <folder> is a sub-folder, one level deep)

<https://containers.dev/implementors/spec/#devcontainerjson>

In fact, we can create demos that include links to multiple Codespaces in a single repository, such as:
<https://github.com/eitsupi/devcontainer-try-r>

**Comment by mcanouil** [comment]
Closing in favour of:
- #6717
